src/styles/main-styles.scss

```diff
@@ -1,15 +1,39 @@
-$color-grey: #a3a3a3;
-$color-layout: rgba(171, 171, 171, 0.55);
-$color-dark-brown: #601919;
-
-body {
-    width: 99%;
-}
-
 .additional-margin-top {
     margin-top: 90px;
 
     &--less {
         margin-top: 45px;
     }
 }
+
```

Comment:
Better would be splitting that styles into small components which refers to react components